Dfe.Academies.External.Web.UnitTest/Pages/WhatAreYouApplyingToDoModelTests.cs

```diff
@@ -55,7 +55,7 @@ public async Task WhatAreYouApplyingToDoModel___OnGetAsync___InValid()
     }
 
     [Test]
-    public async Task WhatAreYouApplyingToDoModel___OnPostAsync___ModelIsValid___InValid()
+    public async Task OnPostAsync___ModelIsValid___InValid()
```

Comment:
I don't understand this test from the title and with a conditional assert it's hard to make out what should be happening.